kymatio/__init__.py

```diff
@@ -13,7 +13,7 @@
             ]
 
 from .scattering1d.scattering1d import Scattering1D
-from .scattering2d.scattering2d import Scattering2D
+from .scattering2d import Scattering2D
```

Comment:
I did this so that the tests pass. Do we want to do this some other way?

Reply:
I'm good with it.